src/main/kotlin/com/jetbrains/snakecharm/inspections/SmkLambdaRuleParamsInspection.kt

```diff
@@ -43,14 +45,28 @@ class SmkLambdaRuleParamsInspection : SnakemakeInspection() {
                                 )
                             }
                             if (pyParameter.name != WILDCARDS_LAMBDA_PARAMETER) {
-                                registerProblem(
-                                        pyParameter,
-                                        SnakemakeBundle.message(
-                                                "INSP.NAME.only.these.parameters.in.section",
-                                                WILDCARDS_LAMBDA_PARAMETER,
-                                                SnakemakeNames.SECTION_INPUT
-                                        )
-                                )
+                                if (index == 0) {
+                                    registerProblem(
+                                            pyParameter,
+                                            SnakemakeBundle.message(
+                                                    "INSP.NAME.only.these.parameters.in.section",
+                                                    WILDCARDS_LAMBDA_PARAMETER,
+                                                    SnakemakeNames.SECTION_INPUT
+                                            ),
+                                            ProblemHighlightType.WARNING,
+                                            null,
+                                            PyRenameArgumentQuickFix()
```

Comment:
Doesn't work properly, doesn't rename usages in the right part of lambda. Although standard rename refactoring for the same element also renames all usages.

Reply:
E.g. `outdir=lambda wd, output:  wd.foo`